### PR TITLE
Update Readme file

### PR DESCRIPTION
**Pull request by TapishKhari**
Update Readme file
1. Added the link of the deployed pages in Readme file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 ## Link to the live page
 
-> Feel free to check the []().
+> Feel free to check the [](https://tapishkhari.github.io/FirstDevProject/).
```
Comment (prolajumokeoni) [inline README.md]: - [ ] Kindly add a title `[live demo](https://tapishkhari.github.io/FirstDevProject/)` so your link is visible and clickable [live demo](https://tapishkhari.github.io/FirstDevProject/) 👍🏿

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @TapishKhari, 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  Good design



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (TapishKhari) [reply README.md]: @prolajumokeoni : I have made the changes as per the requirement.

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**